src/Telemetry/UserLevelCacheWriter.cs

```diff
@@ -39,13 +46,13 @@ public string RunWithCache(string cacheKey, Func<string> getValueToCache)
                     || ex is PathTooLongException
                     || ex is IOException)
                 {
+                    this._telemetryClient.TrackException(ex);
```

Comment:
Does the actual exception message get sent?

Reply:
(followed up separately)